backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/ProofTemplateController.java

```diff
@@ -70,7 +76,12 @@ public HttpResponse<List<ProofTemplate>> listProofTemplates() {
      */
     @Get("/{id}")
     public HttpResponse<ProofTemplate> getProofTemplateForId(@PathVariable UUID id) {
-        return HttpResponse.ok(proofTemplateManager.findProofTemplate(id).toRepresentation());
+        Optional<BPAProofTemplate> proofTemplate = proofTemplateManager.getProofTemplate(id);
```

Comment:
This is not in synch with the main branch. Please use what is there, this is outdated. Also this does not compile!!!

Reply:
Yes, i did a mistake when solving the merge conflict. Didn't recognize that there was an update in the getProofTemplateForId.